app/controllers/profiles/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Profiles::SessionsController < Devise::SessionsController
+# before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  def new
+    super
```

Comment:
Good catch! Just to clarify, these were generated by Devise on creating controllers at this custom route (i.e. profile) most likely for developers' convenience if they want to make custom overrides to default methods. I've removed all of the empty methods for now as we can always consult documentation if we need to customize them in future.